test/insane.js

```diff
@@ -61,6 +61,11 @@ test('only returns tags in the whitelist even if disallowed tag is nested', func
   t.end();
 });
 
+test('allows blank attribute', function (t) {
```

Comment:
```suggestion
test('ignores blank attribute', function (t) {
```

Reply:
What about valid HTML5 like `<input disabled>`?

Reply:
Great point. I added some test cases to show that these are still allowed based on the `allowedAttributes` setting.

This PR's added logic only adds to the `if (lkey === 'class' && attrsOk.indexOf(lkey) === -1) {` case, so I'm not worried about impacting other attributes.